zingolib/src/wallet/transactions/trait_inputsource.rs

```diff
@@ -0,0 +1,139 @@
+use orchard::note_encryption::OrchardDomain;
```

Comment:
IIRC (If I Recall Correctly) this has moved, so this feature probably needs to merge in latest dev.